css/index.css

```diff
@@ -1,3 +1,375 @@
+	height: 100px;
+	position: sticky;
+  top: 0;
+  margin: 0 auto;
+  padding: 10px;
+	z-index: 2;
+  background: white;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  line-height: 36px;
+  margin: 0 10px;
+}
+
+header nav a {
+  margin-right: 70px;
+	text-decoration: none;
+	font-size: 1.12rem;
+  font-weight: bold;
+	color: gray;
+}
+
+header nav a:hover {
+	text-decoration: underline;
+}
+
+header #logo-image-wrapper {
+  width: 210px;
+  margin-right: 20px;
+  justify-self: flex-end;
+  align-self: center;
+}
+
+section {
+  line-height: 1.5;
+}
+
+section h2 {
+  font-size: 1.2rem;
+}
+
+.banner {
+	display: flex;
+	justify-content: flex-end;
+  align-items: center;
+  margin: 60px auto;
+}
+
+.banner .banner-column {
+	flex: 0 0 40%;
+}
+
+.banner #headline {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+  margin: 0 25px;
+	text-align: center;
+  line-height: 1;
+}
+
+.banner #headline h1 {
+	font-size: 5rem;
+}
+
+.banner #headline .button {
+	width: 190px;
+	padding: 10px;
+  border: 1px solid black;
+  font-size: 1.15rem;
+	cursor: pointer;
+}
+
+.banner #hero-image-wrapper {
+  margin: 0 25px;
+}
+
+.main-content-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 991px;
+  padding: 25px 5px;
+  margin: 10px auto;
+}
+
+article {
+  border-top: 2px solid black;
+  border-bottom: 2px solid black;
+  padding: 30px 0;
+}
+
+.introduction {
+	display: flex;
+}
+
+.introduction .intro-column {
+	flex: 0 0 48%;
+}
+
+.middle-image-wrapper {
+  margin: 35px auto;
+	text-align: center;
+}
+
+.product-and-services {
+	display: flex;
+}
+
+.product-and-services .p-and-s-column {
+	flex: 0 0 33%;
+}
+
+.contact-info {
+  padding: 50px 0;
+  line-height: 1;
+}
+
+.contact-info a {
+	color: black;
+	text-decoration: none;
+}
+
+.contact-info a:hover {
+	text-decoration: underline;
+}
+
+.footer {
+  height: 150px;
+	bottom: 0;
+	text-align: center;
+  font-size: 1.1rem;
+}
+
+/* Services Page */
+.services-into-text h1 {
+  margin: 1rem 0;
+  font-size: 2rem;
+}
+
+.services-list {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+  border-top: 1px solid black;
+  margin: 2rem auto;
+}
+
+.service-item {
+	flex: 0 0 48%;
+	border: 1px solid black;
+	padding: 2rem;
+	margin: 1.5rem 0;
+	background: #F4F3F4;
+}
+
+.svc-button {
+	width: 130px;
+	padding: 0.25rem 2rem;
+	margin-top: 1rem;
+	border: 1px solid black;
+	border-radius: 5px;
+	text-align: center;
+	background: white;
+	font-size: 0.75rem;
+	cursor: pointer;
+}
+
+.about-services {
+	display: flex;
+	justify-content: space-between;
+	border-top: 2px solid black;
+}
+
+.about-services-text ul {
+	list-style-type: disc;
+	margin: 1.15rem;
+}
+
+.about-services-text, .services-info-img {
+	flex: 0 0 48%;
+	margin: 1.5rem 0;
+}
+
+
+/*ADAPTIVE DESIGN SETTINGS*/
+/* Tablet Screen */
+@media (max-width: 800px) {
+  .container, .main-content-wrapper {
+    max-width: 800px;
+  }
```

Comment:
When you're starting to add media queries, you should definitely start bringing in PerfectPixel, or some other tool. You're going to want to get the header/nav set as close to pixel-perfect as possible, because the rest of the page's layout flows from the header. I find it's easiest, if I have a design file that I'm asked to match, to just have it overlaying my browser window from the very beginning. So once the html is all hammered out and you start adding styles, get that PerfectPixel fired up right away.

Reply:
Just to clarify when I'm aligning my header using PerfectPixel. My header should line-up with the design image when X and Y coordinates are at zero?

Reply:
Yes using the viewport and side-docking the devtools in conjunction with PerfectPixel is much better than just using Chrome's devtools and screen simulator.

Too bad design files aren't always the actual size in the real world.

Thanks Joe!